### DISCUSSION TITLE
ClassIsland 社区宣传片「课岛，进化。」绝赞完工——

### DISCUSSION THREAD

**Discussion by LyCecilion** [ClassIsland/ClassIsland#342] [展示台]
王婆卖瓜，自卖自夸。

本人对自己的 PPT 水平还是有一定把握的 ~~虽然也是咸鱼一条~~，打算给 ClassIsland 做一个宣传片。不过零音喜欢绝对的神秘感，所以在这里发一个 discussion 作为预告。（

**Comment by LyCecilion** [comment]
不是，你宣传片做哪去了.png

**Reply by jiangyin14** [reply]
不是，你宣传片做哪去了.png

**Reply by MKStoler4096** [reply]
不是，你宣传片做哪去了.png

**Reply by Griseo-bh3rd** [reply]
不是，你宣传片做哪去了.jpeg

**Reply by wang-lu-yuan** [reply]
不是，你宣传片做哪去了.webp

**Reply by Pro-Qin** [reply]
不是，你宣传片做哪去了.html

**Reply by Cartethyia-0612** [reply]
[不是，你宣传片做哪去了.url](https://b23.tv/Ibfabtb)

**Comment by LyCecilion** [comment]
一切都如梦一般恍惚。回过头看的时候，校一模已经过去了 81 天。

今天是高考的 200 天倒计时。

哎，零音又能说什么呢……只好祝我们都能如愿吧。

![mmexport1731768386571.jpg](https://github.com/user-attachments/assets/fb5e4208-25dd-47c6-b22c-f0c73816bcfe)

**Comment by LyCecilion** [comment]


**Reply by jiangyin14** [reply]
赛博诗人.jpg

**Comment by LyCecilion** [comment]
# 宣传片「课岛进化」脚本

> [!WARNING]
>
> 该脚本于早期构思时使用，请以实际宣传片内容为准。

（列举 ClassIsland 为传统课程表方案解决的痛点）

（大标题，超级大）开启课程表智能的新篇章

（ClassIsland 加载动画）

以自动化的方式更新课表（档案编辑界面，展示一下多周轮换之类的高档功能）

使用组件搭配你的主界面（主界面组件军火展示）甚至多行组件（突然拆下来一行吓死观众哈哈哈）

可以自定义位置的窗口（窗口乱动，这里就要用到 LiPolymer 的插件了）确保教学工作的正常进行

还有超级强大的提醒功能（都展示一遍，各种提醒提供方，强调动画之类）

And introducing 🤓 自动化

以及 😎 插件系统（这里就不得不宣传一下 ExtraIsland 了）

换课？调课？不在话下（狠狠抄 CW 宣传片）

（正在开发的）主题系统焕然一新（放用户截图）

极其活跃的开发社群（这里要不要放 hub 呢）

（Ending）课程表——再一次进化

（谢幕）

**Reply by jiangyin14** [reply]
喔↑喔→喔↑哇↓哇↑哇→↑

**Comment by DryIce-cc** [comment]
@all-contributors please add @LyRin-owo for promotion.

**Reply by DryIce-cc** [reply]
ok 看来确实不会理我（）

**Comment by Griseo-bh3rd** [comment]
@DryIce-cc 其实不是不理你，是只处理issue或pr中的at (doge)

**Reply by DryIce-cc** [reply]
> @DryIce-cc 其实不是不理你，是只处理issue或pr中的at (doge)

草（）是我疏忽了，感谢提醒 Σ(¯□¯||)

**Comment by Cartethyia-0612** [comment]
[【全自动？多课表？多功能教学课表软件的再次「进化」| ClassIsland 宣传 PV-哔哩哔哩】]( https://b23.tv/Ibfabtb)
